cortex/serviceconnector.py

```diff
@@ -187,8 +187,9 @@ class _Client:
     URIs = {}
 
     def __init__(self,  *args, **kwargs):
+        # See kubernetes client for better handling of this ..  patch_namespaced_custom_object_with_http_info()  using locals() and checks..
         url = kwargs.get("url")
-        version = kwargs.get("version")
+        version = kwargs.get("version", 4)
```

Comment:
Constants.default_api_version ? Can we use this instead of hardcoding

Reply:
Was just defaulting, can look at re-adding a constant...  the use of @property was causing serialization issues...